### ISSUE TITLE
Quirk in what is selected after coming out of Lettering

### ISSUE THREAD

**Issue by tyrosinase** [inkstitch/inkstitch#3229]
This is a pretty minor thing so if there's no control over it at the plug-in level it's not a big deal, but:

![image](https://github.com/user-attachments/assets/692faa92-c889-4520-9db6-78849a9d37b0)

After opening an existing lettering and making changes to it, the lettering is selected after the dialog closes... but it's actually the objects one level down. So if you do a Duplicate to make another word with the same settings, what you get doesn't carry over the annotations and so Ink/Stitch won't recognize the new object as Lettering.

**Comment by kaalleen** [comment]
You are right. We have no influence on the selection on plugin level. Except if we use dbus (as in select embroidery elements). But that is a bit problematic, so I rather not.

**Comment by tyrosinase** [comment]
Yep, it's not worth messing with if we have to go to that level.
